### PR TITLE
blog for podman container|image exists

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Is this the correct blog?

I read the title, of the PR to state that you are blogging on new commands.

podman container exists
and 
podman image exists

But the blog talks about podman built into images?

File: _posts/2018-11-17-podman-exists.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+title: Build Podman RPMs with a container image
```
Comment (TomSweeneyRedHat) [inline _posts/2018-11-17-podman-exists.md]: looks like a cut/paste leftover from your previous blog.

**Comment by baude** [comment]
@rhatdan @TomSweeneyRedHat updated, some sort of git hiccup.

**Comment by rhatdan** [comment]
LGTM

File: _posts/2018-11-27-podman-exists.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+tags: podman, containers
+---
+
+![podman logo](https://podman.io/images/podman.svg)
+
+{% assign author = site.authors[page.author] %}
+# Podman container|image exists
+## By {{ author.display_name }} [GitHub](https://github.com/{{ author.github }}) [Twitter](https://twitter.com/{{ author.twitter }})
+
+We are seeing a proliferation of Podman usage in users' daily workflows.  As such, these workflows are often scripted -- in something like bash -- and clear exit codes from the applications being run are paramount.  One of the tasks we often see is a user wanting to verify if an image or a container exists in local storage.  We saw several different approaches approaches to solving this including running podman ps or podman images with filters or complex uses of grep.
```
Comment (TomSweeneyRedHat) [inline _posts/2018-11-27-podman-exists.md]: suggest: addi back ticks around `podman ps` and `podman images`.
```
running `podman ps` or `podman images` with
```

File: _posts/2018-11-27-podman-exists.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+### Solution
+
+After a bit of discussion with our users, recorded in [issue #1845] (https://github.com/containers/libpod/issues/1845), a plan was hatched to have a specific command that satisfies this use case.  It was implemented for both containers and images; and I suppose if users wish, we could implement it for pods as well. If the image or container exists, Podman will return an exit code of `0`. Ff it does not exist, Podman will return an exit code of `1`. Any other exit code can be attributed to non-verification failures like permissions or failure in reading local storage.
```
Comment (TomSweeneyRedHat) [inline _posts/2018-11-27-podman-exists.md]: 'Ff it does not exist,' to 'If it does not exist,'

File: _posts/2018-11-27-podman-exists.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+![podman logo](https://podman.io/images/podman.svg)
+
+{% assign author = site.authors[page.author] %}
+# Podman container|image exists
+## By {{ author.display_name }} [GitHub](https://github.com/{{ author.github }}) [Twitter](https://twitter.com/{{ author.twitter }})
+
+We are seeing a proliferation of Podman usage in users' daily workflows.  As such, these workflows are often scripted -- in something like bash -- and clear exit codes from the applications being run are paramount.  One of the tasks we often see is a user wanting to verify if an image or a container exists in local storage.  We saw several different approaches approaches to solving this including running podman ps or podman images with filters or complex uses of grep.
+
```
Comment (TomSweeneyRedHat) [inline _posts/2018-11-27-podman-exists.md:18]: Please add this tag:
```
<!--readmore-->
```
here, and I just realized that's not documented on the README.md and needs to be.  I'll fix that.

**Review by TomSweeneyRedHat — COMMENTED** [review]
One suggestion, two quick fixes, otherwise a nice article @baude, tyvm!

**Comment by baude** [comment]
should be fixed up!

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Going to merge this now.  I'll add a "new" entry for it, feel free to social media this as you wish.

**Pull request merged**